.github/workflows/gen-sui-upgrade-version-pr.yml

```diff
@@ -4,25 +4,30 @@ run-name: Generate Sui Version Upgrade PR
 on:
   workflow_dispatch:
     inputs:
-      new_tag:
+      new_sui_tag:
         description: "New Sui tag (e.g., testnet-v1.55.0), if empty will default to current sui testnet version"
         required: false
         type: string
+      bump_walrus_version:
+        description: "Create Walrus version bump PR"
+        required: false
+        type: boolean
+        default: false
```

Comment:
In my opinion, these two PRs should be created separately: The Sui bump should happen after a Sui testnet release, the version bump after the branch cut. These are typically a few days apart.